src/js/brim/time.js

```diff
@@ -41,12 +48,28 @@ function time(val: Ts | Date = new Date()) {
       return brim.time(ts)
     },
 
+    addDur(dur: Ts) {
```

Comment:
Can we call this `addTs` to match `subTs` below.

Reply:
I can do that for now, but I would say this function is more for adding a duration to a timestamp- it would be kind of weird to add two actual timestamps togther. My use of the `Ts` type here is a little hacky- it was just easy because the payload for `ts` and `dur` in backend is the same (this is because the both are fundamentally int64s of nanoseconds and need to be split up because javascript numbers will lose precision) Probably the actual type here should be `bigint` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt). Excitingly this is now part of the builtin js api. Not-excitingly flow doesn't yet support this (thus why I had to import `big-integer`).

All that said, seeing as you and mason are the vanguards of this codebase, and I but a humble visitor from abroad, to your good judgment I will defer.